facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -112,10 +115,9 @@ trait FapiFrontPress extends Logging {
     val stopWatch: StopWatch = new StopWatch
 
     val pressFuture = getPressedFrontForPath(path)
-      .map { pressedFront: PressedPage =>
-        val json: String = Json.stringify(Json.toJson(pressedFront))
-        FaciaPressMetrics.FrontPressContentSize.recordSample(json.getBytes.length, new DateTime())
-        putPressedJson(path, json)
+      .map { pressedFronts: PressedPageVersions =>
+        putPressedPage(path, pressedFronts.full, FullType)
+        putPressedPage(path, pressedFronts.lite, LiteType)
```

Comment:
what happen if one of the put fails?

Reply:
Then the two pressed files will not match and the error will be logged.

This shouldn't be a problem. It's already possible for the pressed page to be updated between the browser retrieving the initial pressed page, and the show-more request.